pkg/util/kernel/version.go

```diff
@@ -26,8 +26,22 @@ var hostVersion Version
 
 // String returns a string representing the version in x.x.x format
 func (v Version) String() string {
-	a, b, c := v>>16, v>>8&0xff, v&0xff
-	return fmt.Sprintf("%d.%d.%d", a, b, c)
+	return fmt.Sprintf("%d.%d.%d", v.Major(), v.Minor(), v.Patch())
+}
+
+// Major returns the major number of the version code
+func (v Version) Major() uint32 {
```

Comment:
I think the return type should by `byte` or `uint8`, since that is the maximum size of each part.

Reply:
Good point, just pushed a commit with this change